backoffice/backoffice/workflows/api/views.py

```diff
@@ -116,10 +119,14 @@ def create(self, request):
         logger.info("Creating workflow with data: %s", request.data)
         serializer = self.serializer_class(data=request.data)
         if serializer.is_valid(raise_exception=True):
-            workflow = Workflow.objects.create(
-                data=serializer.validated_data["data"],
-                workflow_type=serializer.validated_data["workflow_type"],
-            )
+            logger.info("Validating data against given schema: %s", request.data)
+            validation_errors = list(get_validation_errors(request.data.get("data")))
```

Comment:
wouldn't this raise `ValidationError`? :)

Reply:
ValidationErrors are no real exceptions, just normal objects, e.g. if there are any, the dont get not risen

Reply:
Just to be align with here https://github.com/inspirehep/backoffice/pull/115/files#diff-6c0810e9e873f380de45f0b1ec1906261c199d18654c21bcf3f1efa38aa66ef8R315 that we are catching them